src/utils/dom.ts

```diff
@@ -185,7 +187,7 @@ function querySingle<T extends Element = Element>(
   match = options && options.direct === false ? match : match?.parentElement === element ? match : null;
 
   const selector = options?.selector;
-  return selector ? match.querySelector<T>(selector) : match;
+  return selector ? match?.querySelector<T>(selector) : match;
```

Comment:
Since the default slot selector will also return children in a custom element without a `<slot>`, this should also make sure that `assignedSlot` from the default slot selector result is not null.

Reply:
I don't think the `assignedSlot` will work on the spec test. I think we would have to setup an e2e test?